### PR TITLE
Bug/file-encoding-and-terminal-output-bug

### PR DESCRIPTION
**Pull request by initd1**
- [x] added file encoding,

- [x] fixed typo in config name
- [x] error displaying terminal output when script is run

### CODE REVIEW

File: Email/email_reputation_checker.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def checkEmailBreach(self, email):
         except Exception as e:
             utils.error_message(e.text)
         if response.status_code == 404:
-            logger.debug("{}".format(colored("No breaches found for this email","green")))
+            logging.debug("{}".format(colored("No breaches found for this email","green")))
```
Comment (initd1) [inline Email/email_reputation_checker.py:57]: Fixes https://github.com/initd1/HackAlert/pull/24#issuecomment-1501174640

Comment (GH-Syn) [reply Email/email_reputation_checker.py:57]: Nice work. Is this PR ready for review?

File: main.py [hunk 2]
```diff
@@ -1,7 +1,6 @@
 import argparse
 import json
-import logging
-import logging
+import logger
```
Comment (initd1) [inline main.py:3]: Fixes terminal output: https://github.com/initd1/HackAlert/pull/24#issuecomment-1501174640

**Review by GH-Syn — APPROVED** [review]

File: Email/email_reputation_checker.py (hunk 1 above)
Comment (initd1) [reply Email/email_reputation_checker.py:57]: yes. But I couldn't fix the issue of not logging to traceback.log.  Please create a separate issue if you are able to look into it. thanks

**Comment by initd1** [comment]
Confirmed execution of script:
image

**Pull request merged**